extensions/wikia/EnableDiscussions/EnableDiscussionsController.class.php

```diff
@@ -5,13 +5,14 @@ class EnableDiscussionsController extends \WikiaController {
 	const ENABLE_DISCUSSIONS_NAV = 'wgEnableDiscussionsNavigation';
 	const ENABLE_DISCUSSIONS = 'wgEnableDiscussions';
 	const ENABLE_FORUMS = 'wgEnableForumExt';
+	const ARCHIVE_WIKI_FORUMS = 'wgArchiveWikiForums';
 
 	const P_SITE_ID = 'siteId';
 	const P_ROLLBACK = 'rollback';
 	const REASON = 'Enabling discussions!';
 
 	public function index() {
-		$this->assertCanAccessController();
+		//$this->assertCanAccessController();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this commented out code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS125)